### ISSUE TITLE
Error lack of GPS Data?

### ISSUE THREAD

**Issue by dani2007** [dronemapper-io/NodeMICMAC#34]
Postet this first at the ODM community help Forum, but probably it better fits here. 
I tried to process a dataset on the NodeMicMac, but unfortunately it was not succesful. 

WEBODM tells me: “An orthophoto could not be generated. To generate one, make sure GPS information is embedded in the EXIF tags of your images, or use a Ground Control Points (GCP) file”

but the Images are from a DJI Mavic and do have GPS Information, and from what i found out so fare, GCP are not yet impementet on WebODM for NodeMICMAC. 

[console.txt](https://github.com/dronemapper-io/NodeMICMAC/files/3545969/console.txt)

**Comment by dani2007** [comment]
Thank you very much for the answer. I now did upload my dataset here: 

https://drive.google.com/open?id=1ac5AwGTMg7H1Y77vvNqRDuLCtmY9pMDJ

However I did check the GPS Data from the images and the coordinates are different in every Foto. I loaded them via Geosetter in to google earth to check. (see Images below)
![Anmerkung 2019-08-28 193223](https://user-images.githubusercontent.com/20115705/63878917-d5db3300-c9ca-11e9-84c5-0556d88d7a75.png)
![Anmerkung 2019-08-28 193122](https://user-images.githubusercontent.com/20115705/63878918-d673c980-c9ca-11e9-8575-516ae2296110.png)

By "dublicate geotags" you mean that 2 pictures have the same Geotags, right?

**Comment by dronemapper-io** [comment]
Yes, the geotags look fine on closer inspection. We'll take a look at the data and try to process/recommend optimized settings

**Comment by dronemapper-io** [comment]
I attempted to process the data and wasn't successful. Usually for long transects MM requires at least 4 flight lines to generate a good result. I also noticed that image 0001 and 0002 are disconnected from the scene. I'll give it another shot as time permits over the weekend. Thanks

**Comment by dani2007** [comment]
Okey, thank you very much. Yes, Image 1 and 2 are from a different Flight, but from the same area. I was able to process the images in pix4d, dronedeploy and odm, so It would be interesting comoare the results for me.
